src/components/DepositWidget/Styled.tsx

```diff
@@ -38,53 +27,49 @@ export const InnerWrapper = styled.div`
     text-align: center;
   }
 
-  ul {
+  .WalletItemContainer {
+    display: grid;
+    grid-template-rows: repeat(2, auto) 20px auto;
+    justify-content: stretch;
     align-items: center;
-    list-style: none;
-    text-align: left;
-    margin: auto;
-    padding: 1rem 0 1rem 3rem;
-    max-width: 364px;
 
-    @media only screen and (max-width: 420px) {
-      padding: 1rem 0;
-    }
+    font-weight: 800;
```

Comment:
Any reason for being this specific instead of **bold** or **bolder**

Reply:
Short answer: no. Will change tomorrow, it's a good idea I didn't even think about it that way